webapp/client/public/css/components.css

```diff
@@ -401,18 +404,16 @@ a.back-link:focus a.back-link:active{
 .back-link-icon {
     --size: var(--back-link-size);
     content: url("/icons/arrow_back.svg");
-    margin: 3px 3px 3px 0px;
-
+    margin-right: 8px;
+    margin-top: -1px;
```

Comment:
Out of curiosity: why do we need this?

Reply:
Because of using the icon in different ways in old and new pages. If we haven't this rule it is still jumping a little bit.